.gitignore

```diff
@@ -15,6 +15,7 @@ __pycache__
 docs/_build/
 docs/src/generated/
 docs/src/ssg_execution_times.rst
+docs/src/sg_execution_times.rst
```

Comment:
did not get any ssg_* file but was getting some sg_* files

Reply:
it was a typo error.